hw/ip/otp_ctrl/doc/_index.md

```diff
@@ -25,30 +301,495 @@ The following table lists the main parameters used throughout the OTP controller
 
 Localparam     | Default (Max)         | Top Earlgrey | Description
 ---------------|-----------------------|--------------|---------------
+**TODO add**   |                       |              |
+
 
 ### Signals
 
 {{< hwcfg "hw/ip/otp_ctrl/data/otp_ctrl.hjson" >}}
 
-Signal                  | Direction        | Type                      | Description
-------------------------|------------------|---------------------------|---------------
+The table below lists other OTP controller signals.
+**TODO: this will likely have to be adjusted as the implementation progresses**
+
+Signal                   | Direction        | Type                        | Description
+-------------------------|------------------|-----------------------------|---------------
+`otp_csrng_o`            | `output`         | `otp_csrng_req_t`           | Entropy request to [CSRNG]({{< relref "hw/ip/csrng/doc" >}}).
+`otp_csrng_i`            | `input`          | `otp_csrng_rsp_t`           | Entropy acknowledgement from [CSRNG]({{< relref "hw/ip/csrng/doc" >}}).
+`pwr_otp_init_i`         | `input`          | `pwrmgr_pkg::pwr_otp_req_t` | Initialization request coming from power manager  **TODO: link to power manager docs**
+`pwr_otp_init_o`         | `output`         | `pwrmgr_pkg::pwr_otp_rsp_t` | Initialization response going to power manager  **TODO: link to power manager docs**
+`otp_pwr_state_o`        | `output`         | `otp_pwr_state_t`           | Current OTP state, used by power manager in order to determine whether a shutdown needs to be aborted due to an ongoing OTP operation.
```

Comment:
Do you want to leave a note here that this MAY not be needed?

Reply:
good idea - added a TODO for that.